src/dmd/mtype.d

```diff
@@ -4154,6 +4154,7 @@ extern (C++) final class TypeFunction : TypeNext
     ubyte iswild;               // bit0: inout on params, bit1: inout on qualifier
     Expressions* fargs;         // function arguments
     int inuse;
+    bool incomplete;            // return type or default arguments removed
```

Comment:
I don't really get what this means.

Reply:
Wait until I mark this PR ready for review, I found an issue which may result in removing that flag.

Reply:
What do you suggest would be more descriptive.